x-pack/plugins/cases/public/components/cases_context/cases_context_reducer.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Could you please add a test for the reducer?

Reply:
I usually don't do test for reducer unless they contain business logic. This reducer is merely bypassing the props that come to it. The test will be rather trivial and to a point where it would be testing an implementation detail. What do you think?

Reply:
My only concern is to guard against accidental changes in the actions. For example, changing by mistake the `isFlyoutOpen` to false in the `OPEN_CREATE_CASE_FLYOUT` action. Other than that I am fine.

Reply:
There are worst situations that can happen if you go around changing values from false to true :D